man/fi_opx.7.md

```diff
@@ -51,10 +51,11 @@ Additional features
   and *counters*.
 
 Progress 
-: Only *FI_PROGRESS_MANUAL* is supported.
+: *FI_PROGRESS_MANUAL* and *FI_PROGRESS_AUTO* are supported, for best performance, use 
+  *FI_PROGRESS_MANUAL* when possible. *FI_PROGRESS_AUTO* will spawn 1 thread per CQ.
 
 Address vector 
-: Only the *FI_AV_MAP* address vector format is supported.
+: *FI_AV_MAP* and *FI_AV_TABLE* are both supported. *FI_AV_UNSPEC* is default.
```

Comment:
The FI_AV_UNSPEC part is unnecessary. Alternatively, something like this is more useful: `if *FI_AV_UNSPEC* is specified, *FI_AV_xxx* is used by default`.

Reply:
We'll make this change.

Reply:
The new wording just says that MAP is default without mentioning FI_AV_UNSPEC.